internal/backend/runtime/omni/controllers/omni/schematic_configuration.go

```diff
@@ -198,7 +201,7 @@ func (helper *schematicConfigurationHelper) reconcile(
 		machineExtensionsStatus.Metadata().Labels().Delete(omni.LabelMachineSet)
 	}
 
-	machineExtensionsStatus.TypedSpec().Value.TalosVersion = talosVersion
+	machineExtensionsStatus.TypedSpec().Value.TalosVersion = "v" + talosVersion
```

Comment:
Note: this is a bugfix. Without this, the "Update Extensions" screen on the frontend could not load the list of available extensions.